app/controllers/wishlists_controller.rb

```diff
@@ -0,0 +1,20 @@
+class WishlistsController < ApplicationController
+  def new
+    @wishlist = Wishlist.new
+  end
+
+  def create
+    @wishlist = Wishlist.new(wishlist_params)
+    if @wishlist.save
+      redirect_to new_wishlist_path, notice: "Wishlist was successfully created!"
```

Comment:
This notice string should be loaded from the i18n files, a not exists as a literal in controller.